examples/kepler-integration/README.md

```diff
@@ -27,4 +36,8 @@ export MapboxAccessToken=<your_mapbox_token>
 
 ```sh
 yarn start-local
-```
\ No newline at end of file
+```
+
+## TODO
+- Allow users to select two points instead of a fixed distance
```

Comment:
I don't think we want todo's in this, but fine to note in this issue's description.

Reply:
How about future improvements or roadmap?

Reply:
Removed.